wasp/apps/morse.py

```diff
@@ -0,0 +1,147 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+# Copyright (C) 2021 Francesco Gazzetta
+
+"""Morse translator and notepad
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This app is a simple morse translator that also doubles as a notepad.
+Swipe up for a line, swipe down for a dot, tap for end letter, double tap for
+end word.
+Up to 7 lines of translation will be displayed on a 240x240 screen, and old
+lines will be deleted.
+There is a preview of the next letter at the bottom of the screen.
+
+
+.. figure:: res/MorseApp.png
+    :width: 179
+
+.. note::
+
+    Due to it's niche purpose this app is not included by default in the flash
+    images. It must be separately installed and enabled.
```

Comment:
Oh oops, this is a leftover. As discussed in IRC, I'm including it in the default image. I'll delete the note when I add the images